src/main/java/org/sefglobal/scholarx/util/EmailUtil.java

```diff
@@ -1,42 +1,27 @@
 package org.sefglobal.scholarx.util;
 
-import org.sefglobal.scholarx.model.Email;
-import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.mail.javamail.MimeMessageHelper;
-import org.springframework.stereotype.Component;
-import org.thymeleaf.context.Context;
-import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.sefglobal.scholarx.model.Mail;
+import org.sefglobal.scholarx.service.MailConnection;
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.api.mailer.Mailer;
+import org.simplejavamail.email.EmailBuilder;
 
-import javax.mail.MessagingException;
-import javax.mail.internet.MimeMessage;
-import java.nio.charset.StandardCharsets;
-
-
-@Component
 public class EmailUtil {
+    private final MailConnection mailConnection;
 
-    private final JavaMailSender emailSender;
-
-    private final SpringTemplateEngine templateEngine;
-
-    public EmailUtil(JavaMailSender emailSender, SpringTemplateEngine templateEngine) {
-        this.emailSender = emailSender;
-        this.templateEngine = templateEngine;
+    public EmailUtil(MailConnection mailConnection) {
+        this.mailConnection = mailConnection;
     }
 
-    public void sendSimpleMessage(Email email) throws MessagingException {
-        MimeMessage message = emailSender.createMimeMessage();
-        MimeMessageHelper helper = new MimeMessageHelper(message, MimeMessageHelper.MULTIPART_MODE_MIXED_RELATED, StandardCharsets.UTF_8.name());
-
-        Context context = new Context();
-        context.setVariables(email.getProps());
-
-        String html = templateEngine.process("scholarx", context);
```

Comment:
This templateEngine was used for generating the email content using the template we have in [src/main/resources/templates/scholarx.html ](https://github.com/sef-global/scholarx/blob/development/src/main/resources/templates/scholarx.html) file. If we don't use it, it will send only the `message` string we define in the ProgramUtil.java file.

Reply:
It should be fixed. Thanks @Piumal1999

Reply:
Thank you. The template is working now